lib/cisco_node_utils/feature.rb

```diff
@@ -108,5 +108,23 @@ def self.vn_segment_vlan_based_enabled?
     end
 
     # ---------------------------
+    def self.fabric_frwd_anycast_mac
+      config_get('feature', 'fabric_frwd_anycast_mac')
+    end
+
+    def self.fabric_frwd_anycast_mac=(val)
+      puts 'S:Entered Setter:fabric_frwd_anycast_mac='
+      if val
+        state = ''
+      else
+        state = 'no'
+        val = ''
+      end
+      config_set('feature',
+                 'fabric_frwd_anycast_mac', state, val)
```

Comment:
Is this a sub-feature of feature fabric forwarding or a standalone feature? 
If standalone then fine; if not then this method should first call `Feature.fabric_forwarding_enable` before it tries to enable `fabric_frwd_anycast_mac`.